spec/integration/exporter_spec.rb

```diff
@@ -46,12 +46,10 @@
         user: {
           "contact_groups": [
             {
-              "name":             "generic",
-              "chat_enabled":     false
+              "name":             "generic"
             },
             {
-              "name":             "Work",
-              "chat_enabled":     false
+              "name":             "Work"
```

Comment:
Layout/AlignHash: Align the elements of a hash literal if they span more than one line.